packages/accordion/src/Accordion.ts

```diff
@@ -83,8 +93,20 @@ export class Accordion extends SpectrumElement {
         });
     }
 
-    private handleSlotchange(): void {
+    private handleSlotchange({}: Event & { target: HTMLSlotElement }): void {
         this.focusGroupController.clearElementCache();
+        this.items.forEach((item) => {
+            item.size = this.size;
+        });
+    }
+
+    protected override updated(changed: PropertyValues<this>): void {
+        super.updated(changed);
+        if (changed.has('size')) {
```

Comment:
If we don't apply a size when `m` (unless removing another value) then the attribute shouldn't be sprouted either, which is better for performance.
```suggestion
        if (changed.has('size') && (!!changed.get('size') || this.size !== 'm')) {
```